moped-etl/prefect/moped_test_create_flow.py

```diff
@@ -375,10 +377,6 @@ def drain_service(slug):
 
 
 if __name__ == "__main__":
-    branch = "refactor-user-activation-and-main"
-
+    # this could be another place to change for a test flow, project_name="Moped staging"
     test_commission.register(project_name="Moped")
     # test_decommission.register(project_name="Moped")
-
-    # test_commission.run(branch=branch, database_seed_source="staging")
-    # test_decommission.run(branch=branch)
```

Comment:
since we are running from prefect cloud, I felt it was safe to remove these lines. lemme know if you disagree